azure-pipelines.yml

```diff
@@ -11,30 +11,41 @@ pool:
 
 variables:
   imageName: 'manage-courses-backend-poc'
+  dockerOverride: 'docker-compose -f docker-compose.yml -f docker-compose.az.yml'
```

Comment:
I'm not sure I understand what this does, which might be fine till I need to look after it :-)
Perhaps a comment on the line above explaining what it's for and why it's there, no need to repeat things I can read in the docs, though a link might be handy.

Reply:
Happy to link to the docs. This will go away when travis goes though.

Reply:
https://github.com/DFE-Digital/manage-courses-backend/pull/133/commits/15104ec9004137b8cee9aee2acbb7fc65b7b4ce4